### ISSUE TITLE
Feedback: Mod is Amazing

### ISSUE THREAD

**Issue by Rebel459** [EmilsGithub/Archeology-Plus#2]
Well, you don't have a discord it seems, so I thought I'd leave some positive feedback here: Archeology Plus is amazing!

Just tried out the 2.0 beta, and wow, I can't wait for the full release. You've absolutely nailed it - the structures are incredibly well built and vanilla-friendly, the textures and new items are well thought out, and you've absolutely nailed the balancing! The items are crazy fun but not overpowered, and the spawn rates of the structures ensures they aren't too hard to find while being unobtrusive.

Keep up the good work, this is absolutely the best archeology/1.20-related mod I've found thus far (as someone who's tested way too many) by a mile!

**Comment by EmilsGithub** [comment]
Wow thanks! Reading this really put a smile on my face. 😄 

I mostly put it as a beta cause of balancing / spawn rates so hearing that someone likes them makes me happy.
Still need to figure out how to actually give the player the sigils though, that and some more structures are planned before a full release so look forward to it. 😃

**Comment by Rebel459** [comment]
Yeah, sigils are a tough one to work out how to make them attainable, for sure. I feel just hiding something like that in your average suspicious sand or whatnot would feel a little abrupt, but I’d consider maybe linking them to structures? Eg one could be found either in a chest or suspicious sand in one of the underground rooms of the desert temple (because it’s quite large and the vibe is just perfect for one imo). Idk whether you’d randomise which sigil it is, or have each one exclusive to a specific structure. Also wouldn’t need them to have a 100% spawnrate imo, because if you always know what you’re going to find, that’s less exciting imo. The fire resistance sigil, for example, would work well in a desert structure (or even a nether one), but having multiple possible ones in a pool would work too, idk what you’d do tbh.

On a side note, while it’s a bit less vanilla-y, The Creepers Code had a cool nether trail-ruins-like archeological structure, maybe that’s something you want to explore in the future? Though, again, that’s a bit of a different vibe to what you’ve done so far, but you could always make a nether addon or something ig. 

Also, if you ever want help with balancing or testing, I’m happy to download a dev build and spend some time testing it if that would help you at all.

TLDR idk what I’m saying lol, you’ve probably got better ideas (as you’ve proven already, easily the best 1.20 and archeology mod), just throwing around some thoughts. Super hyped to see what you come up with!

**Comment by EmilsGithub** [comment]
Yeah great idea, I was planning on adding more rooms for the temple anyways.

As for testing the Dev builds, I do have a Discord server, just haven't added it yet. It is linked on modrinth if you wanna hop on and shoot me a DM. Not much work has been done since the beta release though.
